lib/house.rb

```diff
@@ -0,0 +1,72 @@
+class House
+attr_reader :price,
+            :address,
+            :rooms
+
+  def initialize(price, address)
+    @price = (price.delete"$").to_i
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room_to_be_added)
+    @rooms << room_to_be_added
+  end
+
+  def above_market_average?
+    true if @price > 500000
+  end
```

Comment:
This method will return either `true` or `nil`.  It probably makes more sense for it to return either `true` or `false`